### PR TITLE
fix: handle unavailable marks

### PR DESCRIPTION
**Pull request by MayaGillilan**
Open question: What do you think about the name `unavailableMarks`? Another suggestion was `restrictedMarks`

### CODE REVIEW

**Review by DanweDE — APPROVED** [review]
I think the approach makes sense if this will effectively remove any sub/sup we get in through copy/paste if the new prop is set accordingly. Would be great to get a test to see how it'll take effect when you pass sub/sup.

File: packages/rich-text/src/plugins/Text/createTextPlugin.ts [hunk 1]
```diff
@@ -65,6 +65,19 @@ export function createTextPlugin(): RichTextPlugin {
 
       return editor;
     },
+    normalizer: [
+      {
+        match: () => Text.isText,
```
Comment (z0al) [inline packages/rich-text/src/plugins/Text/createTextPlugin.ts:70]: Shouldn't this be `match: Text.isText` ?

File: packages/rich-text/src/RichTextEditor.tsx [hunk 2]
```diff
@@ -140,6 +144,7 @@ const RichTextEditor = (props: Props) => {
             onAction={onAction}
             isDisabled={disabled}
             onChange={setValue}
+            unavailableMarks={unavailableMarks}
```
Comment (z0al) [inline packages/rich-text/src/RichTextEditor.tsx:147]: I think `restrictedMarks` may be more descriptive but have no strong opinion. This is a public prop so let's spend a few minutes thinking about a good name

**Review by z0al — COMMENTED** [review]
Looks good overall

File: packages/rich-text/src/RichTextEditor.tsx (hunk 2 above)
Comment (giotiskl) [reply packages/rich-text/src/RichTextEditor.tsx:146]: 👍🏻 to what Ahmed said, I'd suggest `restrictedMarks` or perhaps `prohibitedMarks`

Comment (DanweDE) [reply packages/rich-text/src/RichTextEditor.tsx]: `forbiddenMarks`, `marksYouShallNotPass` :mage_man: :laughing:

**Pull request merged**